index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+</head>
+<body>
+  <div class="header">
+    <div class="welcome miss-desktop">Welcome</div>
+    <nav class="nav">
+      <a href="#" class="menu miss-desktop"><i class="material-symbols-rounded">menu</i></a>
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>  
+  </div>
+  <main>
+    <section class="headline">
+      <div class="item">
+        <h2 class="intro">Hey there, I'm Harriet</h2>
+        <h2 class="intro green">I'm a software developer</h2>
+      </div>
+      <p class="item"> 
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socials item">
+        <i class="fab fa-brands fa-twitter"></i>
+        <i class="fab fab-brands fa-linkedin-in"></i>
+        <i class="fab fa-brands fa-github"></i>
+        <i class="fa fa-regular fa-hand-peace"></i>
+      </div>
```

Comment:
- [ ] Kindly I recommend you to use `ul` tag to group them, and `li` per icon; each icon has a link `a` tag, Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using `href`.

Reply:
completed